press/index.md

```diff
@@ -7,82 +7,6 @@ redirect_from:
-
-![Purple square](http://d1zymp9ayga15t.cloudfront.net/images/CP-purp.gif) 2008: Anne Carpenter's talk as part of a series that overviews Broad Institute research in non-technical terms, "Broad 101" [[.MOV](http://d1zymp9ayga15t.cloudfront.net/content/movies/Broad101-AnneCarpenter.mov)]
-
-![Red square](http://d1zymp9ayga15t.cloudfront.net/images/CP-red.gif) 2008: Public television special featuring Anne Carpenter and the CellProfiler team: Bold Visions: Women in Science and Technology [[YouTube link (abridged version)](http://www.youtube.com/watch?v=Hw7_Fxvktf0)]
-
-![Red square](http://d1zymp9ayga15t.cloudfront.net/images/CP-red.gif) 2007: Article mentioning CellProfiler in The Scientist, "Lab Tools: Seeing Faster, Seeing Smarter" [[PDF](http://d1zymp9ayga15t.cloudfront.net/content/papers/ECallaway_LabTools_TheScientist-2007.pdf)]
-
-![Purple square](http://d1zymp9ayga15t.cloudfront.net/images/CP-purp.gif) 2007: Anne Carpenter's talk to girls at the Museum of Science Women in Research day [[.MOV](http://d1zymp9ayga15t.cloudfront.net/content/movies/WomenInResearch.mov)] [[RealPlayer](http://d1zymp9ayga15t.cloudfront.net/content/movies/WomenInResearchVideo.rm)]
-
-![Red square](http://d1zymp9ayga15t.cloudfront.net/images/CP-red.gif) 2007: Profile of the CellProfiler project team in Biotechniques [[PDF](http://d1zymp9ayga15t.cloudfront.net/content/papers/26.5-BiotechAboutAuthors.pdf)]
-
-![Red square](http://d1zymp9ayga15t.cloudfront.net/images/CP-red.gif) 2006: Article about CellProfiler project co-founder, Anne Carpenter in Genome Technology: "Imaging Breakthrough by Necessity" [[PDF](http://d1zymp9ayga15t.cloudfront.net/content/papers/25.5GenomeTech.pdf)]
-
-![Purple square](http://d1zymp9ayga15t.cloudfront.net/images/CP-purp.gif) 2006: Slides from Anne Carpenter's presentation about CellProfiler [[PDF](http://d1zymp9ayga15t.cloudfront.net/content/papers/CellProfilerHCAPresentation.pdf)]
-
-![Red square](http://d1zymp9ayga15t.cloudfront.net/images/CP-red.gif) 2005: Article about launch of CellProfiler in BioInform [[PDF](http://d1zymp9ayga15t.cloudfront.net/content/papers/BioInformInterview.pdf)]
-
-![Purple square](http://d1zymp9ayga15t.cloudfront.net/images/CP-purp.gif) 2004: Anne Carpenter's talk at Society for Biomolecular Screening conference in Orlando [[.MOV](http://d1zymp9ayga15t.cloudfront.net/content/movies/2006-SBS-Conference-Anne.mov)[](http://d1zymp9ayga15t.cloudfront.net/content/movies/2006-SBS-Conference-Anne.mov)]
-
-![Red square](http://d1zymp9ayga15t.cloudfront.net/images/CP-red.gif) 2004: Interview of Anne Carpenter on high-content screening in Inside BioAssays [[PDF](http://d1zymp9ayga15t.cloudfront.net/content/papers/InsideBioassaysInterview.pdf)]
+You can see talks and read articles about CellProfiler and the group at the Carpenter Lab website [[here](https://personal.broadinstitute.org/anne/press.html)].
```

Comment:
This goes to carpenter lab press page, which says to refer to this page to learn about our Press! Let's have it just live in one place. Update this once we decide, Anne will make the final call

Reply:
There should just be one Press page, and it should live in Carpenter lab from now on. @rvale17